COMMERCE-9961 - Product API does not accept expiration date in the past

victorcdp opened the pull request:
Related issue: [COMMERCE-9961](https://issues.liferay.com/browse/COMMERCE-9961)

Problem was that there was a check to see if the product's expiration date was before the current one.
Proposed fix is to remove that and allow any expiration date as long as it is after the product's display date.

brianikim commented:
ci:test:relevant

brianikim commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest(master)/7488/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-20.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7488/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-7.liferay.com/job/test-portal-source-format/1563/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-7.liferay.com//userContent/jobs/test-portal-source-format/builds/1563/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 3 minutes 6 seconds 21 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/a0efb8c930d3246f80cc349dba4f0c85be8a4768">a0efb8c930d3246f80cc349dba4f0c85be8a4768</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/katienesterovich/liferay-portal/tree/LPS-102715">LPS-102715</a><br/>Branch GIT ID: <a href="https://github.com/katienesterovich/liferay-portal/commit/efd6fb5ccaef369132a482190ab7cd7fc3ab69b8">efd6fb5ccaef369132a482190ab7cd7fc3ab69b8</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-7.liferay.com/job/test-portal-source-format/1563/"><strike><strong>test-portal-source-format</strong></strike></a></li></ul></div><h5>For more details click <a href="https://test-1-7.liferay.com/userContent/jobs/test-portal-source-format/builds/1563/jenkins-report.html">here</a>.</h5><pre><code>format-source-all:
 [sync-dir] Synchronizing /opt/dev/projects/github/liferay-binaries-cache-2020/.gradle/caches/modules-2/files-2.1 into /opt/dev/projects/github/liferay-portal/.gradle/caches/modules-2/files-2.1
 [sync-dir] 0 files synchronized in 326ms
     [exec] Using proxy squid.lax.liferay.com:3128.
     [exec] Configuration on demand is an incubating feature.
     [exec] 
     [exec] &gt; Task :yarnCheckFormat
     [exec] Running the Yarn "checkFormat" script
     [exec] 
     [exec] &gt; Task :downloadNode SKIPPED
     [exec] 
     [exec] &gt; Task :yarnCheckFormatModules
     [exec] yarn run v1.13.0
     [exec] \$ liferay-npm-scripts check --production false
     [exec] apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js: BAD
     [exec] Prettier checked 1 file, 1 file has problems
     [exec] 
     [exec] FAILURE: Build failed with an exception.
     [exec] 
     [exec] * What went wrong:
     [exec] Execution failed for task ':yarnCheckFormatModules'.
     [exec] &gt; java.io.IOException: Process '[/opt/dev/projects/github/liferay-portal/build/node/bin/node, /opt/dev/projects/github/liferay-portal/modules/yarn-1.13.0.js, checkFormat, --production, false]' finished with non-zero exit value 1
     [exec] 
     [exec] * Try:
     [exec] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':yarnCheckFormatModules'.
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:110)
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:77)
     [exec] 	at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)</code></pre></details></html>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 24 out of 26 jobs passed in 1 hour 16 minutes 38 seconds 400 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/a0efb8c930d3246f80cc349dba4f0c85be8a4768">a0efb8c930d3246f80cc349dba4f0c85be8a4768</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/744c27e5da71a70ce9c0cbf7b1546f66d33aa66a">744c27e5da71a70ce9c0cbf7b1546f66d33aa66a</a></p>24 out of 26 jobs PASSED<div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest(master)/7488/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/323069/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</strong></strike></a></li></ul></div><details><summary><strong>24 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/321941/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/288537/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/323070/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/326394/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/282488/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/345003/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/317358/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/322603/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/316752/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/267792/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/6</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/345002/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/326393/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/196814/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/272375/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/317357/">test-portal-acceptance-pullrequest-batch(master)/js-unit-private-jdk8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/316751/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/339611/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/327216/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221421/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/267791/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/321942/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/339612/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/327215/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/81977/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-20.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7488/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/323069/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/323069//consoleText">AXIS_VARIABLE=0,label_exp=!master #323069</a><div><p>Please fix <strong>semantic versioning</strong> on <strong><a href="https://github.com/katienesterovich/liferay-portal/tree/LPS-102715">katienesterovich/LPS-102715</a></strong><pre><code>     [exec] Note: Some input files use unchecked or unsafe operations.
     [exec] Note: Recompile with -Xlint:unchecked for details.
     [exec] 
     [exec] &gt; Task :apps:dynamic-data-mapping:dynamic-data-mapping-form-web:compileJava
     [exec] 
     [exec] &gt; Task :apps:dynamic-data-mapping:dynamic-data-mapping-form-web:copyLibs SKIPPED
     [exec] &gt; Task :apps:dynamic-data-mapping:dynamic-data-mapping-form-web:classes
     [exec] Note: Some input files use or override a deprecated API.
     [exec] Note: Recompile with -Xlint:deprecation for details.
     [exec] Note: Some input files use unchecked or unsafe operations.
     [exec] Note: Recompile with -Xlint:unchecked for details.
     [exec] 
     [exec] &gt; Task :apps:dynamic-data-mapping:dynamic-data-mapping-web:compileJSP
     [exec] 
     [exec] &gt; Task :apps:dynamic-data-mapping:dynamic-data-mapping-web:autoUpdateXml SKIPPED
     [exec] &gt; Task :apps:dynamic-data-mapping:dynamic-data-mapping-web:jar
     [exec] &gt; Task :apps:dynamic-data-mapping:dynamic-data-mapping-web:baseline SKIPPED
     [exec] &gt; Task :apps:dynamic-data-mapping:dynamic-data-mapping-web:syncVersions
     [exec] 
     [exec] &gt; Task :apps:dynamic-data-mapping:dynamic-data-mapping-form-web:generateJSPJava
     [exec] Compiling JSP files of project ':apps:dynamic-data-mapping:dynamic-data-mapping-form-web' with /opt/dev/projects/github/liferay-portal/bundles/tomcat-9.0.17/webapps/ROOT/WEB-INF/lib/util-taglib.jar as dependency in place of 'com.liferay.portal:com.liferay.util.taglib'
     [exec] Note: Some input files use or override a deprecated API.
     [exec] Note: Recompile with -Xlint:deprecation for details.
     [exec] Note: Some input files use unchecked or unsafe operations.
     [exec] Note: Recompile with -Xlint:unchecked for details.
     [exec] 
     [exec] &gt; Task :apps:dynamic-data-mapping:dynamic-data-mapping-form-web:compileJSP
     [exec] 
     [exec] FAILURE: Build failed with an exception.
     [exec] 
     [exec] * What went wrong:
     [exec] Execution failed for task ':apps:dynamic-data-mapping:dynamic-data-mapping-api:baseline'.
     [exec] &gt; Semantic versioning is incorrect while checking /opt/dev/projects/github/liferay-portal/tools/sdk/dist/com.liferay.dynamic.data.mapping.api-6.0.0.jar against /opt/dev/projects/github/liferay-portal/.gradle/caches/modules-2/files-2.1/com.liferay/com.liferay.dynamic.data.mapping.api/5.6.0/3cf0e2b6207768bf7737c1dd46ddc32a42690469/com.liferay.dynamic.data.mapping.api-5.6.0.jar</code></pre></p></div></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">here</a>.</h4></details></html>

brianikim commented:
hey Katie, I don't think this is the right way to approach the solution. Looking at the documentation for AlloyUI ImageViewer, https://alloyui.com/tutorials/image-viewer/, it looks like there are existing implementation of the close button specifically provied by AlloyUI. I would look into why this is not being present for our reproduction steps.

Commit:
LPS-169826 Return null instead of blank string

ccorreagg committed:
LPS-173905 Add reference in Portlet as the onAfterSave method of the listener is not called until the configuration is saved for the second time if configurationPolicy = REQUIRE is set

julschong committed:
LPS-172618 Remove Stream usages from UserReindexerPerformanceOfLargeUserGroupInManySitesTest

julschong committed:
LPS-172618 Cleanup unused fields and change _getDocument return type to void since it is never used

julschong committed:
LPS-172618 Remove Stream usages from UserGroupCascadeReindexUsersTest

Commit:
LPS-169826 Set listTypeDefinitionId to ObjectField when importing an Object Definition

julschong committed:
LPS-172618 Remove Stream usages from UserIndexerIndexedFieldsByAssociationTest, use List.toString() and List.sort() instead of Stream

JanisZhang committed:
LPS-171132 Remove Optional usages in DataDefinitionContentTypeRegistry

JanisZhang committed:
LPS-171132 Remove Optional usages in DataDefinitionResourceImpl

JanisZhang committed:
LPS-171132 Remove Stream usages in DataDefinitionResourceImpl

JanisZhang committed:
LPS-171132 Remove Stream usages in DataDefinitionDDMFormUtil

JanisZhang committed:
LPS-171132 Remove Stream usages in DataLayoutUtil

JanisZhang committed:
LPS-171132 Remove Stream usages in DataStorageUtil

sandrodw3 committed:
LPS-173488 Add react component for web content usages

sandrodw3 committed:
LPS-173488 buildLang

julschong committed:
LPS-172618 SF, simplified by inlining _getUserIds() and removing redundant sb operations

david-truong committed:
LPS-173700 cache transformed sql in publications also

balazssk committed:
LPS-172113 Should be false, not select-more-than-one

balazssk committed:
LPS-172113 Follow true-false pattern from JSP

balazssk committed:
LPS-17211 Revert existing new values to the original

Commit:
LPS-172991 Add checkRepetitions function to handle the visibility of the remove-duplicate-field

ruben-pulido committed:
LPS-173580 Create LayoutClassedModelUsagesHelper

ruben-pulido committed:
LPS-173580 Make LayoutClassedModelUsagesHelper available through ServletContextUtil

baharturk committed:
COMMERCE-10657 Add three new functional tests

adolfopa committed:
LPS-70835 Check if a page with same title already exists

After sending a page to trash, it is possible to create a page with
the same title.  As titles must be unique per group, we need to check
for this case before trying to sync using LARs.

Note that staging is not affected by this -- it is not possible to
edit live -- but site templates use LARs to propagate its changes to
sites.

EugenioAntunes committed:
LPS-173785 Create Raylife Data Objects - Sales Goals Inserting Data

ruben-pulido committed:
LPS-173580 Create GetLayoutClassedModelUsagesStrutsAction

ruben-pulido committed:
LPS-173488 Don't use portletNamespace

ruben-pulido committed:
LPS-173580 Use LayoutClassedModelUsagesHelper

ruben-pulido committed:
LPS-173580 Add getUsagesData to LayoutClassedModelUsagesDisplayContext

dantewang committed:
LPS-171132 SF, rename

dantewang committed:
LPS-171132 SF, avoid checking the same condition repeatedly

Commit:
LPS-172991 Fix test: Wrap the PageProvider with FormProvider

ambrinchaudhary committed:
LPS-173880 Separate result's component

ambrinchaudhary committed:
LPS-173880 Show clay empty message

ambrinchaudhary committed:
LPS-173880 buildLang

Commit:
LPS-174017 Fix usage getActionVariables

ambrinchaudhary committed:
LPS-173881 Use a ClayList to show the results

david-truong committed:
LPS-173700 fix tests

Commit:
LPS-174017 Fix integration test

rafaellajordao committed:
LPS-166766 test fix for ContentPerformance#ChangeLanguageInDocumentDisplayPage

rafaellajordao committed:
LPS-166766 test fix for ContentPerformance#ChangeLanguageInWidgetPage

rafaellajordao committed:
LPS-166766 test fix for ContentPerformance#MetricsIconVisibleInBlogDisplayPage

rafaellajordao committed:
LPS-152127 test fix for ContentPerformance#MetricsIconVisibleInContentPage

rafaellajordao committed:
LPS-152127 test fix for ContentPerformance#MetricsIconVisibleInDocumentDisplayPage

rafaellajordao committed:
LPS-152127 test fix for ContentPerformance#MetricsIconVisibleInWidgetPage

Commit:
LPS-172368 Adjust

davisaints committed:
LPS-173939 Fix CanSetPaginationNumeric test

ambrinchaudhary committed:
LPS-173881 Highlight search query in results

ling-alan-huang committed:
LPS-174058 Auto SF

samanzanedo committed:
LPS-174138 Check if there is some JSON data before trying to normalize it

ambrinchaudhary committed:
LPS-172546 Animate as the Clay component

Commit:
LPS-172991 Fix: Add check to make sure there is a fieldName

ealonso committed:
LPS-172368 SF

ealonso committed:
LPS-172368 Rename

ealonso committed:
LPS-172368 Match with _getAssetPriority

Commit:
LPS-174163 Add ApproveEventWithInvitedUser to quaratine

murilostodolni committed:
LPS-174111 Add feature flag to LPS-155659

Commit:
LPS-174168 Add ViewProcessItemDetailPage to quarantine

Commit:
LPS-174178 Add CalendarUpgrade tests to quarantine

sergiojimcos committed:
LPS-155809 else condition is missing

brianchandotcom committed:
LPS-171132 SF

brianchandotcom committed:
LPS-173700 SF

brianchandotcom committed:
LPS-172618 Rename

brianchandotcom committed:
LPS-70835 Wordsmith

JerryAdrianoo committed:
LPS-172299 Fix ObjectRelationships#CanRelateOneToManyCustomObjectEntries

brianchandotcom committed:
LPS-174111 Move package

brianchandotcom committed:
LPS-173880 Wordsmith

brianchandotcom committed:
LPS-173880 Regen

brianikim committed:
COMMERCE-10078 Add keyword support for SKU and Names in Order Search

brianchandotcom committed:
LPS-173580 & LPS-173488 SF

brianchandotcom committed:
COMMERCE-10078 Rename

Commit:
COMMERCE-9961 Make it possible to input product expiration date before current date

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

victorcdp commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/fe66b60dfb479e4bd8428c8bf9eb06d16116a157">fe66b60dfb479e4bd8428c8bf9eb06d16116a157</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/victorcdp/liferay-portal/tree/COMMERCE-9961">COMMERCE-9961</a><br/>Branch GIT ID: <a href="https://github.com/victorcdp/liferay-portal/commit/b83ca785b84a9c134c485c3888fb229606c2d8bc">b83ca785b84a9c134c485c3888fb229606c2d8bc</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-31.liferay.com/job/test-portal-source-format/4631/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-31.liferay.com/userContent/jobs/test-portal-source-format/builds/4631/jenkins-report.html">here</a>.</h5><pre>     [java] java.lang.Exception: Found formatting issues:
     [java] At least one commit message is missing a reference to a required JIRA project: BLADE, CLDSVCS, COMMERCE, FACES, IDE, ISSITES, LCD, LHC, LOOP, LPS, LRAC, LRCI, LRDOCS, LRIS, LRQA, OAUTH2, POSHI, RELEASE, SYNC, TR. Please verify that the JIRA project keys are specifiedin ci.properties in the liferay-portal repository.
     [java]   at com.liferay.source.formatter.util.JIRAUtil.validateJIRAProjectNames(JIRAUtil.java:66)
     [java]   at com.liferay.source.formatter.SourceFormatter._validateCommitMessages(SourceFormatter.java:1278)
     [java]   at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:322)
     [java]   at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:286)
[stopwatch] [run.batch.test.action: 54.658 sec]
     [echo] The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/portal-impl/build.xml:453: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/portal-impl/build.xml:596: Java returned: 1
      [get] Getting: http://test-1-31/job/test-portal-source-format/4631//consoleText
      [get] To: /opt/dev/projects/github/liferay-portal/20230201115303131.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/20230201115303131.txt
  [typedef] Could not load definitions from resource org/apache/maven/artifact/ant/antlib.xml. It could not be found.
  [taskdef] Could not load definitions from resource org/jacoco/ant/antlib.xml. It could not be found.
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null860544945.properties

merge-test-results:
    [mkdir] Created dir: /opt/dev/projects/github/liferay-portal/test-results
[beanshell] Truncating errors in /opt/dev/projects/github/liferay-portal/portal-impl/test-results/TEST-JenkinsLogAssertTest.xml
[junitreport] Processing /opt/dev/projects/github/liferay-portal/test-results/TESTS-TestSuites.xml to /tmp/null28259293
[junitreport] Loading stylesheet jar:file:/opt/java/ant/lib/ant-junit.jar!/org/apache/tools/ant/taskdefs/optional/junit/xsl/junit-frames.xsl
[junitreport] Transform time: 157ms
[junitreport] Deleting: /tmp/null28259293
     [echo] A report with all the test results can be found at test-results/html/index.html.
    [mkdir] Created dir: /opt/dev/projects/github/liferay-jenkins-ee/test-results</pre></details></html>

victorcdp commented:
ci:test:sf

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-31.liferay.com/job/test-portal-source-format/4631/">test-portal-source-format#4631</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-31.liferay.com/userContent/jobs/test-portal-source-format/builds/4631/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/brianikim/liferay-portal/pull/54">brianikim#54</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2183181250">[master] ci:test:sf - victorcdp &gt; brianikim - PR#54 - 2023-02-01[11:50:27]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-31.liferay.com/job/publish-testray-report/17170/">publish-testray-report#17170</a><br/></div></div>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/fe66b60dfb479e4bd8428c8bf9eb06d16116a157">fe66b60dfb479e4bd8428c8bf9eb06d16116a157</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/victorcdp/liferay-portal/tree/COMMERCE-9961">COMMERCE-9961</a><br/>Branch GIT ID: <a href="https://github.com/victorcdp/liferay-portal/commit/5a654d0c05885a06dc6172e718e68ed9336690af">5a654d0c05885a06dc6172e718e68ed9336690af</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-33.liferay.com/job/test-portal-source-format/1617/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-33.liferay.com/userContent/jobs/test-portal-source-format/builds/1617/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-33.liferay.com/job/test-portal-source-format/1617/">test-portal-source-format#1617</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-33.liferay.com/userContent/jobs/test-portal-source-format/builds/1617/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/brianikim/liferay-portal/pull/54">brianikim#54</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2183181495">[master] ci:test:sf - victorcdp &gt; brianikim - PR#54 - 2023-02-01[11:55:59]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-33.liferay.com/job/publish-testray-report/6539/">publish-testray-report#6539</a><br/></div></div>
